update description (#1)
DevExpress-Examples/asp-net-web-forms-grid-implement-popup-edit-form-in-batch-mode: Readme.md

# Grid View for ASP.NET Web Forms - How to implement a popup edit form in batch mode
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/t286169/)**
<!-- run online end -->

This example demonstrates how to use a popup control to edit grid data in batch mode.

![Popup Edit Form](popupEditForm.png)

## Overview

> Follow the steps below:

Reviewer: To do what?

Author: ```suggestion
Follow the steps below to implement a popup edit form in batch mode:
```

1. Create the [Grid View](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) control and set its [Mode](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridViewEditingSettings.Mode) property to `Batch`.

    ```aspx
    <dx:ASPxGridView ID="ASPxGridView1" ClientInstanceName="grid" runat="server" ... >
        <ClientSideEvents BatchEditStartEditing="onStartEdit" />
        <SettingsEditing Mode="Batch" />
        <!-- ... -->
    </dx:ASPxGridView>
    ```

> 2. Create a popup control and fill its content with child controls.

Reviewer: ```suggestion
2. Create a popup control and add child controls.
```
